src/spyglass/position/v1/position_trodes_position.py

```diff
@@ -267,7 +278,7 @@ def make(self, key):
                 .replace(" valid times", "")
             )
             + 1
-        )
+        )  # TODO: Fix this hack
```

Comment:
There are quite a few places in the code where we substring `pos X valid times` - def worth revising/unifying the approach
- `position_dlc_selection.pyL333`
- `common_position.pyL544`
- `common_interval.pyL133` uses regex, which is a little better
- `common_behav.py.PositionSource` has some helper funcs for compose/decompose

Reply:
agreed.  The correct solution now would use a join of `PositionIntervalMap * TaskEpoch` which is what I do in my personal code.  Would make sense to put this in a common function